hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/io/storage/HoodieSparkFileReaderFactory.java

```diff
@@ -33,6 +33,7 @@ protected HoodieFileReader newParquetFileReader(Configuration conf, Path path) {
     conf.setIfUnset(SQLConf.PARQUET_INT96_AS_TIMESTAMP().key(),
         SQLConf.PARQUET_INT96_AS_TIMESTAMP().defaultValueString());
     conf.setIfUnset(SQLConf.CASE_SENSITIVE().key(), SQLConf.CASE_SENSITIVE().defaultValueString());
+    conf.setIfUnset("spark.sql.legacy.parquet.nanosAsLong", "false");
```

Comment:
nit: Can we add a comment to explain why we put a plain string here?

Reply:
Sure I can add a comment before this